accounts-cluster-bench/src/main.rs

```diff
@@ -475,10 +476,14 @@ fn run_accounts_bench(
                 }
             }
 
-            if close_nth > 0 {
-                let expected_closed = total_accounts_created as u64 / close_nth;
-                if expected_closed > total_accounts_closed {
-                    let txs: Vec<_> = (0..expected_closed - total_accounts_closed)
+            if close_nth_batch > 0 {
+                let expected_closed = (total_accounts_created as u64
+                    / (close_nth_batch * batch_size as u64))
+                    * batch_size as u64;
```

Comment:
```suggestion
                let num_batches_to_close = total_accounts_created as u64
                    / (close_nth_batch * batch_size as u64);
                let expected_closed = num_batches_to_close * batch_size as u64;
```
Total nit, but just makes it a little more clear why the `batch_sizes` don't cancel out

Reply:
Done!